virtcontainers/bridgedmacvlan_endpoint.go

```diff
@@ -115,3 +116,52 @@ func (endpoint *BridgedMacvlanEndpoint) HotAttach(h hypervisor) error {
 func (endpoint *BridgedMacvlanEndpoint) HotDetach(h hypervisor, netNsCreated bool, netNsPath string) error {
 	return fmt.Errorf("BridgedMacvlanEndpoint does not support Hot detach")
 }
+
+func (endpoint *BridgedMacvlanEndpoint) save() (s persistapi.NetworkEndpoint) {
+	s.Type = string(endpoint.Type())
+	s.BridgedMacvlan = &persistapi.BridgedMacvlanEndpoint{
```

Comment:
Nit: Would be clearer to add a blank line after `s.Type = ...`.

Reply:
ok